spec/features/admin/configuration/dynamic_content_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+
+# Defines the test link string
+TEST_LINK = "ofn test link"
+
+# Defines the test href string. This one should contain all non-encoding characters, for a URL
+TEST_HREF = ":/?#@!$&'()*+,;=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
I would move these declarations down into the test where they're used, a bit like this:

```ruby
describe 'add dynamic content to homepage' do
  
  ...

  context "editing the content section" do
    let(:test_link) { "ofn test link" }
    let(:test_href) { ":/?#@!$&'()*+,;=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ" }

    ...
    
  end
end
```

Reply:
Will do so in the upcoming specs :+1: